mrp_bom_tracking/views/bom_template.xml

```diff
@@ -0,0 +1,44 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- Copyright 2019 Eficent Business and IT Consulting Services S.L.
+     License LGPL-3.0 or later (https://www.gnu.org/licenses/lgpl). -->
+
+<odoo>
+    <template id="track_bom_line_template">
+        <div>
+            <strong>The component quantities or UoMs have been updated.</strong>
+            <ul>
+                <t t-foreach="lines" t-as="line">
+                    <li><t t-esc="line.product_id.display_name"/>:</li>
+                    Product Quantity: <t t-esc="line.product_qty" /><t t-if='line.product_qty != product_qty'> -&gt; <t t-esc="float(product_qty)"/></t><br/>
+                    Product Unit of Measure: <t t-esc="line.product_uom_id.display_name" /><t t-if='line.product_uom_id != product_uom_id'> -&gt; <t t-esc="product_uom_id.display_name"/></t><br/>
+                </t>
+            </ul>
+        </div>
+    </template>
+
+    <template id="track_bom_template">
+        <div>
+            <strong>The components have changed.</strong>
+            <ul>
+                <t t-foreach="lines" t-as="line">
+                <li><t t-esc="line.product_id.display_name"/>:
+                <t t-esc="mode"/> component<br/></li>
+                </t>
+            </ul>
+        </div>
+    </template>
+
+    <template id="track_bom_template_2">
+        <div>
+            <strong>The components have changed.</strong>
+            <ul>
+                <t t-foreach="lines" t-as="line">
+                <li><t t-esc="line.product_id.display_name"/>:
```

Comment:
It would be nice to have a detail of what line has been modified, something like:
```
Test Subproduct: New/Removed Component
Product Quantity: 1.0 
Product Unit of Measure: Units
```
So if somebody modified a bom line, that information can be displayed:
```
Test Subproduct: Removed component 
Product Quantity: 12
Product Unit of Measure: Units
Test Subproduct: New component
Product Quantity: 1
Product Unit of Measure: Dozens
```

I also miss track on product variant.